doc/storage.md

```diff
@@ -61,7 +61,7 @@ lxc storage volume set [<remote>:]<pool> <volume> <key> <value>
 
 # Storage Backends and supported functions
 ## Feature comparison
-LXD supports using ZFS, btrfs, LVM or just plain directories for storage of images, instances and custom volumes.  
+LXD supports using ZFS, btrfs, LVM or just plain directories for storage of images, instances and custom volumes.
```

Comment:
This entire commit is likely wrong, two whitespace at the end of a line is markdown syntax for a forced line break.
So you're effectively undoing formatting here :)